ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -117,6 +241,12 @@ unsafe fn init(
 ) {
     init_logger();
 
+    if let Err(reason) = redirect_stdout_stderr() {
+        warn!("Error redirecting stdout/stderr: {}", reason);
+    } else {
+        println!("Capi/lib.rs: init() function called redirect_stdout_stderr() successfully.\n");
```

Comment:
I think we can get rid of this println now that we've verified that it works as expected.

Reply:
I thought of that too, but I thought it would be good to have so that the user can verify/look in the VS2007 output window and get a concreate statement that stdout/stderr is being redirected.

I can remove it.

Reply:
If all the new changes look good, I can commit the changes.

Reply:
Yes, it's much easier to review changes as actual comits rather than GitHub comments.

Reply:
Committed latest changes.